packages/embark/src/lib/modules/code_generator/index.js

```diff
@@ -346,20 +353,43 @@ class CodeGenerator {
         code += self.generateStorageInitialization(true);
         code += self.generateNamesInitialization(true);
         code += self.getReloadPageCode();
+        code += "\nexport default EmbarkJS;";
         code += '\n/* eslint-enable */';
 
         next();
       },
       function writeFile(next) {
         self.generateArtifact(code, constants.dappArtifacts.embarkjs, '', next);
+      },
+      function transformCode(artifactPath, updated, next) {
+        if (!updated) {
+          return next();
+        }
+        transform(code, {
+          cwd: self.fs.embarkPath(),
```

Comment:
Maybe this works fine, but changing the working directory makes me a little concerned there could be surprises later on.

For an alternative approach see:
https://github.com/embark-framework/embark/blob/master/packages/embark/src/lib/modules/pipeline/webpack.config.js#L20-L29
https://github.com/embark-framework/embark/blob/master/packages/embark/src/lib/modules/pipeline/webpack.config.js#L70-L77
https://github.com/embark-framework/embark/blob/master/packages/embark/src/lib/modules/pipeline/webpack.config.js#L167

I'm not sure which approach is better, they may have the same consequences.